.pre-commit-config.yaml

```diff
@@ -16,11 +16,15 @@ repos:
       - id: check-added-large-files # Don't accidentally commit giant files.
       - id: check-merge-conflict # Watch for lingering merge markers.
       - id: check-yaml # Validate all YAML files.
+      - id: check-toml
+      - id: check-json
       - id: check-case-conflict # Avoid case sensitivity in file names.
       - id: debug-statements # Watch for lingering debugger calls.
       - id: mixed-line-ending # Use Unix line-endings to avoid big no-op CSV diffs.
         args: ["--fix=lf"]
+      - id: trailing-whitespace
       - id: name-tests-test # Follow PyTest naming convention.
+      - id: end-of-file-fixer
```

Comment:
I got overeager with removing these when integrating `ruff`, and forgot that these fixes apply to **all** file types, not just Python (which `ruff` does address)